index.html

```diff
@@ -9,55 +9,44 @@
 </head>
 <body>
     <section class="header">
-        <nav>
-            <div class="logo"><p>My Logo</p></div>
-            <div class="menu"><img src="img/Union.png"></div>
+    <header>
+        <nav class="MenuBar">
+            <div class="logoPlace"> 
+                <a href="index.html" class="logo">
+                        <p>My Logo</p>
+                </a>
+            </div>
+            <div class="menu">
+                <img src="img/Union.png">
+            </div>
         </nav>
-        <div class="welcome">
-            <p>I’m Rashmi.<br>
-              Glad to see<br> 
-              you!</p>
-        </div>
+    </header>
```

Comment:
- Great Job for fixing previous reviewer sections 👍 I think you missed one thing moving the header outside the section so please fix that for semantic reasons 👍 

check the bellow code snippet 👇
```
<body>
    <header>
        .....
    </header>
    <section>
       ......
    </section>
</body>
```